### PR TITLE
Edited the raw link

### LINKED ISSUE #330: Link not working
in the below section the link is returning 404
> For convenience, we encourage you to copy the vim.json code into your new file, as a starting point. Copy the raw file, as the JSON below won't work due to inline comments. 

"raw" link

### PR DESCRIPTION
**Pull request by gautamkrishnar**
fixes #330

### CODE REVIEW

File: duckduckhack/goodie/goodie_cheat_sheets.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ Below is a summary of the [`vim.json`](https://github.com/duckduckgo/zeroclickin
 
 The above Instant Answer was created by simply adding [`vim.json`](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/share/goodie/cheat_sheets/vim.json), explained below. 
 
-**For convenience, we encourage you to copy the [`vim.json`](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/share/goodie/cheat_sheets/vim.json) code into your new file, as a starting point.** Copy the [raw file](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/share/goodie/cheat_sheets/vim.json), as the JSON below won't work due to inline comments.
+**For convenience, we encourage you to copy the [`vim.json`](https://raw.githubusercontent.com/duckduckgo/zeroclickinfo-goodies/master/share/goodie/cheat_sheets/json/vim.json) code into your new file, as a starting point.** Copy the [raw file](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/share/goodie/cheat_sheets/vim.json), as the JSON below won't work due to inline comments.
```
Comment (moollaza) [inline duckduckhack/goodie/goodie_cheat_sheets.md]: @gautamkrishnar good catch! Though you fixed the wrong link. The next one is supposed to link to the raw text

Comment (moollaza) [inline duckduckhack/goodie/goodie_cheat_sheets.md]: Actually, they both might as well point to the raw file. It's a little confusion that the bolded one doesn't point to what we're hoping the dev will copy.

/cc @talsraviv

**Comment by gautamkrishnar** [comment]
@moollaza sorry for that :smile: 
Fixed all links in the page. 
I am now checking other pages for the same kind of error, if there is any i will open more PRs.
cc: @talsraviv

**Comment by moollaza** [comment]
LGTM :+1:

**Comment by talsraviv** [comment]
Thanks!

**Pull request merged**